src/components/text/index.tsx

```diff
@@ -50,7 +43,7 @@ export type TextProps = RNTextProps &
   };
 export type TextPropTypes = TextProps; //TODO: remove after ComponentPropTypes deprecation;
 
-type PropsTypes = BaseComponentInjectedProps & ForwardRefInjectedProps & TextProps;
+const AnimatedText = Animated.createAnimatedComponent(RNText);
```

Comment:
Isn't this going to cause worse performance, because now, this will be called every single time, when before, only when `Text` is animated